awx/settings/defaults.py

```diff
@@ -599,6 +599,11 @@
 
 VMWARE_VALIDATE_CERTS = False
 
+# -----------------
+# -- VMware ESXi --
+# -----------------
+VMWARE_ESXI_EXCLUDE_EMPTY_GROUPS = True
```

Comment:
@thedoubl3j same here

Reply:
I believe the inventory settings remain canonical. I saw no suggestion otherwise.